taskcat/amiupdater.py

```diff
@@ -330,7 +330,7 @@ def _determine_regions(self):
                 self._regions.add(region)
 
     def set_region_ami(self, cn, region, ami_id):
-        currvalue = self._contents['Mappings']['AWSAMIRegionMap'].get(region, None).get(cn, None)
+        currvalue = self._mapping_root.get(region, None).get(cn, None)
```

Comment:
Shouldn't region default to `{}` otherwise `None.get(cn, None)` will raise an exception

Reply:
Good catch